src/utils/unoserver.ts

```diff
@@ -48,20 +48,26 @@ export class Unoserver {
 	 *
 	 * @param from source file
 	 * @param to target file
+	 * @param filter filter name to use during conversion process
 	 */
-	convert(from: string, to: string): Promise<void> {
+	convert(from: string, to: string, filter?: string): Promise<void> {
```

Comment:
Could you please make the third argument as an options object? Like this:

```ts
from: string,
to: string,
options?: {  filter?: string }
```

Reply:
Done